stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1082,6 +1083,7 @@ public class ChatClient internal constructor(
         return api.sendEvent(eventType, channelType, channelId, extraData)
     }
 
+    @InternalStreamChatApi
     public fun getVersion(): String = VERSION_PREFIX + BuildConfig.STREAM_CHAT_VERSION
```

Comment:
Maybe we can make it `internal`. @JcMinarro what do you think?

Reply:
Is this method usefull for our customers?
I think we are using it into our Sample App to show the version we are using.
Maybe we could keep it public

Reply:
I'll keep it as public w/o the annotation!

Reply:
I think public _with_ annotation is fine as we only want to use it in our own sample, and not have clients using it

Reply:
@JcMinarro @adasiewiczr Should we keep the annotation then? I can see both sides at the moment, but I'll let you folks decide given that you have more experience with this.